README.md

```diff
@@ -185,15 +186,15 @@ class State(rx.State):
 
 The state defines all the variables (called vars) in an app that can change and the functions that change them.
 
-Here the state is comprised of a `prompt` and `image_url`. There are also the booleans `processing` and `complete` to indicate when to show the circular progress and image.
+Here the state is comprised of a `prompt` and `image_url`. There are also the booleans `processing` and `complete` to indicate when to disable the button (during image generation) and when to show the resulting image.
 
 ### **Event Handlers**
 
 ```python
 def get_image(self):
     """Get the image from the prompt."""
     if self.prompt == "":
-        return rx.window_alert("Prompt Empty")
+        return rx._x.toast.error("Prompt Empty")
```

Comment:
You might want to change this back as well so that it uses the window alert instead of the experimental toast

Reply:
done